test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -6034,6 +6034,12 @@ def forward(self, x):
                       dynamic_axes={"x": [1, 2]},
                       test_with_inputs=[y])
 
+    def test_prelu_scalar(self):
```

Comment:
Does this test fail before this change and pass afterwards? If not then I don't think this test is useful.